keyboards/satan/keymaps/admiralStrokers/Makefile

```diff
@@ -0,0 +1,26 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE 	=		no   # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE 	= 	no  # Mouse keys(+4700)
+EXTRAKEY_ENABLE 	=		yes  # Audio control and System control(+450)
+CONSOLE_ENABLE 		=		no   # Console for debug(+400)
+COMMAND_ENABLE 		=		no   # Commands for debug and configuration
+SLEEP_LED_ENABLE	=		no   # Enables your LED to breathe while your computer is sleeping.
+NKRO_ENABLE 			=		yes  # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE 	=		no   # Enable keyboard backlight functionality
+MIDI_ENABLE 			=		no   # MIDI controls
+AUDIO_ENABLE 			=		no   # Audio output on port C6
+UNICODEMAP_ENABLE	=		no	 # This allows sending unicode symbols using X(<unicode>) in your keymap.
+UNICODE_ENABLE 		=		no   # Unicode
+UCIS_ENABLE 			=		no	 # Keep in mind that not all will work (See WinCompose for details on Windows).
+BLUETOOTH_ENABLE 	=		no   # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE 	=		no   # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE	=		no   # Breathing sleep LED during USB suspend
+
+VARIABLE_TRACE		=		no	 # Use this to debug changes to variable values
```

Comment:
Can you try removing this line or the tabs? The tabs here might be messing this up, causing it to turn "on."